src/decompression.jl

```diff
@@ -286,3 +286,96 @@ function decompress_aux!(
     end
     return A
 end
+
+function decompress_aux!(
+    A::AbstractMatrix{R}, B::AbstractMatrix{R}, result::TreeSetColoringResult
+) where {R<:Real}
+    A .= zero(R)
+    S = get_matrix(result)
+    color = column_colors(result)
+    forest = result.tree_set.forest
```

Comment:
can you list the internals of DataStructures.jl that we use while working with their `DisjointSets`?

Reply:
only for future reference if something breaks